crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
> 'squash' the two functions into one to reduce duplicated code

It might be appropriate, but this _asn1_mon controversy should not be the reason. I mean if there are reasons other than that, like improved readability, then yes, it would be sensible thing to do.